pom.xml

```diff
@@ -279,6 +278,29 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>jbpm-installer-exclude-prod-non-productized</id>
+            <activation>
+                <property>
+                    <name>!productized</name>
+                </property>
+            </activation>
+            <modules>
+                <module>jbpm-installer</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>jbpm-installer-exclude-prod-productized-false</id>
+            <activation>
+                <property>
+                    <name>productized</name>
+                    <value>false</value>
+                </property>
+            </activation>
+            <modules>
+                <module>jbpm-installer</module>
+            </modules>
+        </profile>
     </profiles>
```

Comment:
I think both cases/conditions can be covered by simple 
`<value>!true</value>`

Reply:
let me try it

Reply:
@mareknovotny much simpler with
```
<name>productized</name>
<value>!true</value>
```
thanks a lot